### PR TITLE
Add salary range

### PR DESCRIPTION
**Pull request by CoreyAR**
- Add Salary Range 
- Add API_URL to `.env.example`
- Fix filepath in `knexfile.js`

### CODE REVIEW

**Comment by egdelwonk** [comment]
Looks good! 

Any chance you could update the migration to set the lower/upper salary value to the same values based on the salary_range before dropping the column ?

The original salary_range was to have a enum of ranges, as well, which may work.

**Comment by CoreyAR** [comment]
Totally. Before I do that, should I make it an enum of ranges or keep it as is?

**Comment by egdelwonk** [comment]
Ranges are fine as long as we have the migration for existing data.

**Comment by CoreyAR** [comment]
I ended up switching to an enum so that it follows the same pattern as experience range which I think is important for consistency.

I also added a seed for test data.

File: server/controllers/job.js [hunk 1]
```diff
@@ -99,8 +98,7 @@ exports.update = async function(req, res, next) {
   req
     .assert("experience_range", "Please enter a valid experience_range")
     .isInt();
-  req.assert("lower_salary", "Please enter a valid lower_salary").isInt();
-  req.assert("upper_salary", "Please enter a valid upper_salary").isInt();
+  req.assert("salary_range", "Please enter a valid lower_salary").isInt();
```
Comment (egdelwonk) [inline server/controllers/job.js]: Should this read "Please enter a valid salary range."?

Comment (CoreyAR) [inline server/controllers/job.js]: 🙈 Yes, it should.

**Comment by egdelwonk** [comment]
Great! Thanks. We may need to convert the existing jobs salary data to use the new ENUM values. Any ideas?

**Comment by CoreyAR** [comment]
I think that is covered in the migration. The seeder that I added has a salary saved as a dollar value so you could run the seeder, then run the migrations to test. 
https://github.com/egdelwonk/nashdev-jobs/pull/31/files#diff-25d624080caf012072cb280e94b5795a

**Comment by egdelwonk** [comment]
Argh, you are right! So sorry, I overlooked that file. Thank you for doing this!

**Pull request merged**